packages/mobile/src/onboarding/UseBackToWelcomeScreen.tsx

```diff
@@ -52,9 +52,12 @@ export function useBackToWelcomeScreen({ backAnalyticsEvent }: Props) {
     // Check if this screen is removed while focused (hardware back button, swipe, or simply navigate back)
     // so we can reset some state
     // See https://reactnavigation.org/docs/preventing-going-back
-    const cancelBeforeRemove = navigation.addListener('beforeRemove', (e) => {
+    const cancelBeforeRemove = navigation.addListener('beforeRemove', (event) => {
       ValoraAnalytics.track(backAnalyticsEvent)
```

Comment:
This one shouldn't be called if the action is not dispatched.

Reply:
Yup, my bad!